pkgs/applications/kde/ffmpegthumbs.nix

```diff
@@ -1,7 +1,7 @@
 {
```

Comment:
nit: this is libsForQt5.ffmpegthumbs, not kdePackages.ffmpegthumbs.

Reply:
Ah, hence the file path…

FIxed, thank you.